src/lib/singularity.h

```diff
@@ -158,14 +158,4 @@
     extern int singularity_fork_exec(char **argv);
 
 
-
-
-    extern int singularity_config_open(char *config_path);
-    extern void singularity_config_close(void);
-    extern void singularity_config_rewind(void);
-
-    extern char *singularity_config_get_value(char *key);
-    extern int singularity_config_get_bool(char *key, int def);
-
```

Comment:
I think we can have this file include others: I just don't see the value in copy/pasting the same definition twice.  Seems like it'd lead to errors.